documentation/slate/source/includes/objects.md

```diff
@@ -311,17 +522,18 @@ pt-date-time (pt stands for "public transport") is a complex date time object to
 Enum used to identify what kind of objects *[/places](#places)*, *[/pt_objects](#pt-objects)* or *[/disruptions](#disruption)* services are managing.
 
 
-|Value|Description|
-|-----|-----------|
-|administrative_region|a city, a district, a neighborhood|
-|network|a public transport network|
-|commercial_mode|a public transport branded mode|
-|line|a public transport line|
-|route|a public transport route|
-|stop_area|a nameable zone, where there are some stop points|
-|stop_point|a location where vehicles can pickup or drop off passengers|
-|address|a point located in a street|
-|poi|a point of interest|
+| Value                                             | Description                                                   |
+|---------------------------------------------------|---------------------------------------------------------------|
+| [administrative_region](#administrative-region)   | a city, a district, a neighborhood                            |
+| [network](#network)                               | a public transport network                                    |
+| [commercial_mode](#commercial-mode)               | a public transport branded mode                               |
+| [line](#line)                                     | a public transport line                                       |
+| [route](#route)                                   | a public transport route                                      |
+| [stop_area](#stop-area)                           | a nameable zone, where there are some stop points             |
+| [stop_point](#stop-point)                         | a location where vehicles can pickup or drop off passengers   |
+| [address](#address)                               | a point located in a street                                   |
+| [poi](#poi)                                       | a point of interest                                           |
+| [trip](#trip)                                     | a trip                                                        |
```

Comment:
Added note about vehicle_journeys here https://github.com/CanalTP/navitia/pull/1650/commits/8f36ece31e10d874b0be50c49751f9aae0c76747 as there is no link in `links` yet